.travis.yml

```diff
@@ -28,10 +29,14 @@ matrix:
       env: SYMFONY_VERSION="3.2.*"
     - php: 7.1
       env: SYMFONY_VERSION="3.3.*"
+    - php: 7.2
+      env: SYMFONY_VERSION="3.4.*"
+    - php: 7.2
+      env: SYMFONY_VERSION="4.0.*"
```

Comment:
I'm not sure what we'll be able to support Symfony 4.0, maybe we should only support the LTS versions (2.7, 2.8 and 3.4) for the moment.

Reply:
I will play around with it a bit ..
That being said .. some of the DI based features of this Bundle likely need to change for Symfony 4 to really make sense ..

but if we don’t support Symfony 4 .. then all Bundles using our code here will have a problem.

Reply:
You're right. But maybe we can focus this PR on supporting only LTS versions (so that users can use Symfony 2.7, 2.8 or 3.4 today) and work on the Symfony 4.0 support in another PR.